src/drivers/navio_gpio/navio_gpio.cpp

```diff
@@ -0,0 +1,213 @@
+#include <px4_posix.h>
```

Comment:
I mean the PX4 Copyright with the BSD license. We have it in all header/source files.